src/processor/Package.ts

```diff
@@ -10,7 +10,8 @@ import {
   ExportableCombinedCardFlagRule,
   ExportableContainsRule,
   ExportableOnlyRule,
-  ExportableCaretValueRule
+  ExportableCaretValueRule,
+  ExportableRule
```

Comment:
⚠️ ⚠️

Reply:
Since it isn't clear here, I think the lint has a problem because `ExportableRule` is defined but not used.

Reply:
:panic:

Reply:
I didn't realize our lint check on GitHub wouldn't fail if there was a warning. I guess that's probably fair, but this is what I get for forgetting to run it myself. It should be fixed in this very non-suspicious commit 4a3a012.